docs/BUILD.md

```diff
@@ -88,78 +88,78 @@ You won't need to install these for general development but may wish to do so if
 
 There are a few other NPM scripts that can be useful during build and development.
 
-##### `npm start`
+##### `yarn start`
 
 The start command starts up our demo app using the Angular CLI on port 4200 and watches for file changes for live reload.
 
-##### `npm run build`
+##### `yarn build`
 
 This script builds npm package candidates for all four packages we currently publish: `@clr/angular`, `@clr/ui`,
 `@clr/icons`, and `@clr/core` under the `/dist` folder.
 
-##### `npm test` and `npm run angular:test:watch`
+##### `yarn test` and `yarn angular:test:watch`
 
-This entry file now lives in src/clr-angular/test.tsand is now confined to what's under clr-angular. It used to be that the file was a single entry point across multiple packages. You can run the tests in watch mode, so they run continuously `npm run angular:test:watch`.
+This entry file now lives in src/clr-angular/test.tsand is now confined to what's under clr-angular. It used to be that the file was a single entry point across multiple packages. You can run the tests in watch mode, so they run continuously `yarn angular:test:watch`.
 
-##### `npm run build:ci`
+##### `yarn build:ci`
 
 The `build:ci` script is used by Travis-CI to run all of the checks, such as format, lint, and unit tests.
 If the code doesn't pass both the format and lint checkers, then the build fails before running the unit tests.
 
-##### `npm run format` and `npm run format:fix`
+##### `yarn format` and `yarn format:fix`
 
-We use [clang-format](https://github.com/angular/clang-format) as formatter for our code. `npm run format` will not
+We use [clang-format](https://github.com/angular/clang-format) as formatter for our code. `yarn format` will not
 actually format the file but only check if there are any files that would be changed. We do this via a shell script (`./scripts/clang-check.sh`),
 which runs the clang-format command (with `-output-replacements-xml` flag) and greps for any replacement that would be produced.
 
-The `npm run format:fix` does the actual formatting according to the rules specified in `.clang-format` file.
+The `yarn format:fix` does the actual formatting according to the rules specified in `.clang-format` file.
```

Comment:
[nit] I know this was probably a search-n-replace but can we remove this instead? we have not used clang-format in a _long_ time 😳